test/fakeintake/server/serverstore/store.go

```diff
@@ -40,7 +40,7 @@ func NewStore() *Store {
 }
 
 // AppendPayload adds a payload to the store and tries parsing and adding a dumped json to the parsed store
-func (s *Store) AppendPayload(route string, data []byte, encoding string, collectTime time.Time) error {
+func (s *Store) AppendPayload(route string, data []byte, encoding string, collectTime time.Time) {
```

Comment:
since we remove the usage of `tryParseAndAppendPayload()` in this function not need anymore that this function return an error.